3-networks/variables.tf

```diff
@@ -26,7 +26,15 @@ variable "terraform_service_account" {
   description = "Service account email of the account to impersonate to run Terraform."
 }
 
-variable "default_region" {
-  description = "Default subnet region standard_shared_vpc currently only configures one region"
+variable "nat_region" {
+  description = "NAT region standard_shared_vpc"
```

Comment:
Is it possible to provide some sane defaults for these regions? I think overall we should standardize on a default region that can be followed through out.